autovalet_lane_detection/utils/hsv_check.py

```diff
@@ -16,7 +16,7 @@ def nothing(x):
 
 while(1):
 
-    frame = cv2.imread('../data/green/train_image00312.jpg')
+    frame = cv2.imread('/home/evamo0508/Downloads/image.png')
```

Comment:
did linux env variable for $HOME work?

Reply:
this is just for testing, we won't have to run this piece of code eventually, but yea, I updated with the usage of `os.environ.get('HOME')`